tox.ini

```diff
@@ -1,14 +1,24 @@
 [tox]
 skipsdist = True
 minversion = 2.7.0
-envlist = flake8
+envlist = flake8, archivematica
 
-[testenv:flake8]
+[testenv]
 basepython = python2
-skip_install = true
+
+[testenv:flake8]
 deps = flake8
+commands = flake8 .
+
+[testenv:archivematica]
+skip_install = True
+setenv =
+    PYTHONPATH = src/archivematicaCommon/lib:src/dashboard/src
 commands =
-    flake8 .
+    pip install -r src/archivematicaCommon/requirements/test.txt
+    pip install -r src/MCPClient/requirements/test.txt
+    pip install -r src/dashboard/src/requirements/test.txt
+    py.test {posargs}
```

Comment:
`{posargs}` is used here so we can pass arguments to pytest, e.g.:

- `tox -e archivematica -- -x`
- `tox -e archivematica -- src/dashboard`
- [etc...](https://docs.pytest.org/en/latest/usage.html)